tests/unit/test_api_calls.py

```diff
@@ -43,18 +44,16 @@
         (False, "pending", UploadStatus.CANCELLED, CantChangeUploadStatus),
         (False, "uploadable", UploadStatus.UPLOADED, CantChangeUploadStatus),
         (False, "not_uploadable", UploadStatus.UPLOADED, UploadNotRegisteredError),
-        (True, "uploaded", UploadStatus.UPLOADED, RequestFailedError),
+        (True, "uploaded", UploadStatus.UPLOADED, MaxRetriesReached),
     ],
 )
 def test_patch_multipart_upload(
-    bad_url,
-    upload_id,
-    upload_status,
-    expected_exception,
+    bad_url, upload_id, upload_status, expected_exception, max_retries  # noqa: F811
```

Comment:
See above

```suggestion
    bad_url, 
    upload_id, 
    upload_status, 
    expected_exception, 
    max_retries: int,  # noqa: F811
```

Reply:
This gets autoformatted with the comment at the end.
Even if I place this differently, the comment still gets moved to the end and the rest is formatted as above...

Reply:
@MoritzHahn1337 Only way this works is adding another non-empty comment for one of the other arguments

Reply:
Adding type annotations also seems to trigger formatting to use one line for each argument.
I think I'll have another look at the tests and add type annotations where they are missing

Reply:
black autoformats to one line per comment, if you add a comma after the last argument

Reply:
Its always the things you direclty forget ...